packages/mdc-ripple/README.md

```diff
@@ -102,76 +116,79 @@ Mixin | Description
+| `registerDocumentInteractionHandler(evtType: string, handler: EventListener) => void` | Registers an event handler that's invoked when the documentElement is interacted with using type `evtType` |
+| `deregisterDocumentInteractionHandler(evtType: string, handler: EventListener) => void` | Unregisters an event handler that's invoked when the documentElement is interacted with using type `evtType` |
+| `registerResizeHandler(handler: Function) => void` | Registers a handler to be called when the surface (or its viewport) resizes. Our default implementation adds the handler as a listener to the window's `resize()` event. |
+| `deregisterResizeHandler(handler: Function) => void` | Unregisters a handler to be called when the surface (or its viewport) resizes. Our default implementation removes the handler as a listener to the window's `resize()` event. |
+| `updateCssVariable(varName: string, value: (string or null)) => void` | Programmatically sets the css variable `varName` on the surface to the value specified. |
+| `computeBoundingRect() => ClientRect` | Returns the ClientRect for the surface. |
+| `getWindowPageOffset() => {x: number, y: number}` | Returns the `page{X,Y}Offset` values for the window object as `x` and `y` properties of an object (respectively). |
 
-#### MDCRipple.deactivate()
+### MDCRippleFoundation
+
+Method Signature | Description
+--- | ---
+`activate() => void` | Triggers an activation of the ripple (the first stage, which happens when the ripple surface is engaged via interaction, such as a `mousedown` or a `pointerdown` event). It expands from the center.
+`deactivate() => void` | Triggers a deactivation of the ripple (the second stage, which happens when the ripple surface is engaged via interaction, such as a `mouseup` or a `pointerup` event). It expands from the center.
+`layout() => void` | Recomputes all dimensions and positions for the ripple element. Useful if a ripple surface's position or dimension is changed programmatically.
+`setUnbounded(boolean: unbounded) => void` | Sets the ripple to be unbounded or not based on the given boolean.
 
-Triggers a deactivation of the ripple (the second stage, which happens when the ripple surface is engaged via
-interaction, such as a `mouseup` or a `pointerup` event). It expands from the center.
+## Tips/Tricks
 
-#### MDCRipple.layout()
+### Using a sentinel element for a ripple
 
-Recomputes all dimensions and positions for the ripple element. Useful if a ripple surface's
-position or dimension is changed programmatically.
+Usually, you'll want to leverage `::before` and `::after` pseudo-elements when integrating the ripple into MDC-Web components. If you find you can't use pseudo-elements to style the ripple, another strategy could be to use a sentinel element that goes inside your element and covers its surface. Doing this should get you the same effect.
```

Comment:
"If you find you can't use pseudo-elements to style the ripple, another strategy could be to use a sentinel element that goes inside your element and covers its surface."

Too many words!!! Try this:

"If you can't use pseudo-elements, create a sentinel element inside your root element. The sentinel element covers the root element's surface."

also..."Doing this should get you the same effect"...I mean...it better! Just drop that sentence entirely.

...and maybe skim other wordy sections, and find a way to simplify it to active grammar.

Reply:
Done! Also tried to make all other sections (mostly tips and caveats) less wordy too.